setup.py

```diff
@@ -4,7 +4,7 @@
 
 project_url = 'https://github.com/melexis/robot2rst'
 
-requires = ['robotframework<=3.1.2', 'mlx.traceability', 'mako']
+requires = ['robotframework', 'mlx.traceability', 'mako']
```

Comment:
Is this backwards compatible with 3.1?

Reply:
Yes. The old implementation will be used when the new one throws an `ImportError`.